src/app/reviewer/[orgSlug]/study/[studyId]/review/job-review-buttons.tsx

```diff
@@ -45,7 +44,7 @@ export const JobReviewButtons = ({
         },
         onError: reportMutationError('Failed to update study job status'),
         onSuccess: () => {
-            router.push('/')
+            window.location.assign(`/reviewer/${orgSlug}/dashboard`)
```

Comment:
again, don't think we should perform raw navigation, router.push will be much faster